src/sidebar/components/menu-item.js

```diff
@@ -58,51 +59,60 @@ function MenuItem({
   const rightIcon = isSubmenuItem ? renderedIcon : null;
 
   return (
-    <div
-      aria-checked={isSelected}
-      className={classnames('menu-item', {
-        'menu-item--submenu': isSubmenuItem,
-        'is-disabled': isDisabled,
-        'is-expanded': isExpanded,
-        'is-selected': isSelected,
-      })}
-      role="menuitem"
-      {...(onClick && onActivate('menuitem', onClick))}
-    >
-      <div className="menu-item__action">
-        {hasLeftIcon && (
-          <div className="menu-item__icon-container">{leftIcon}</div>
-        )}
-        {href && (
-          <a
-            className={labelClass}
-            href={href}
-            target="_blank"
-            rel="noopener noreferrer"
+    // Wrapper element is a `<div>` rather than a `Fragment` to work around
+    // limitations of Enzyme's shallow rendering.
```

Comment:
I haven't looked deeply into the details, but Enzyme complained when trying to use `wrapper.setProps()` to re-render the component that re-rendering is not supported when there are multiple top-level elements (as is the case when using `Fragment`). Using a concrete element fixes this by making it so that the output of `MenuItem` only has one top-level element.